homework/12_V/1_Aleksandra/homework1/checkIntersection.py

```diff
@@ -0,0 +1,45 @@
+import math
+
+def test_get_intersections():
+    assert get_intersections(2, 2, 1, 6, 2, 3) == (3, 2)
+    assert get_intersections(2, 2, 1, 6, 2, 4) == (2.125, 1.0078432583507784, 2.125, 2.9921567416492216)
+    assert get_intersections(2, 2, 1, 6, 2, 1) == 0
+    assert get_intersections(2, 2, 1, 2, 2, 1) == 1
+    assert get_intersections(6, 2, 3, 6, 2, 2) == 2
+    assert get_intersections(6, 2, 2, 6, 2, 3) == 2
+
+
+def get_intersections(x0, y0, r0, x1, y1, r1):

```

Comment:
The API of this function is not very good. It would be better to return enum or a special result class. Can you refactor the code with to use a result class?

Reply:
Switched it to Enum class.